### DISCUSSION TITLE
Error: Not upgrading 1 `installer manual` cask.

### DISCUSSION THREAD

**Discussion by wagnergerald** [Homebrew/discussions#1514] [Everyday usage] [answered]
When trying to upgrade the `libreoffice-language-pack` package and get the message:

```
$ brew upgrade --cask libreoffice-language-pack 
Error: Not upgrading 1 `installer manual` cask.
libreoffice-language-pack
$ 
```

I'm a bit lost here what an _installer manual_ package is and how to handle it.

Is it [related to this FAQ entry about installer manual](https://docs.brew.sh/Cask-Cookbook#installer-manual)?

**Comment by nandahkrishna** [comment, marked as answer]
Casks with `installer manual` need to be installed by the user using the GUI installer. As a consequence, I don't think Homebrew can upgrade them automatically either. You might have to use `brew reinstall` and run the GUI installer again (I am not a Cask expert though).

**Reply by wagnergerald** [reply]
Thank you for your reply!

I used `brew reinstall --cask libreoffice-language-pack` and the package was updated.

**Reply by Joshua2504** [reply]
Thank you! This also fixes the issue for other apps, e.g. jagex.

**Reply by juandesant** [reply]
Indeed. And the Spanish government's AutoFirma… as long as the certificate is not manually removed. If it is, it is actually better to force install, then ask for the reinstallation.
